src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomPosition = new Random().nextInt(Colors.values().length);
+        return Colors.values()[randomPosition].toString();
```

Comment:
Instead of using `toString()`, use the `name()` method to get the string representation of the enum constant. This is a more reliable approach as per the checklist guidelines.